index.html

```diff
@@ -7,28 +7,24 @@
     <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-    <div id="container" class="container">
-        <div class="title">
-            Seunbayo Test Quiz
-        </div>
-        <div id="question" class="question">
-            Who is the founder of Microsoft
-        </div>
+    <div id="quizcontainer" class="container">
+        <div class="title">Seunbayo Test Quiz</div>
+        <div id="question" class="question"></div>
         <label class="option" for=""> 
```

Comment:
It still doesn't select the answer when you click on the label. I think the empty `for=""` is confusing it. You can remove it. The label around the input is enough for the browser to understand that the label is for that input.